lib/rules/jsx-curly-spacing.js

```diff
@@ -10,13 +10,24 @@
 
 module.exports = function(context) {
   var spaced = context.options[0] === 'always';
+  var multiline = context.options[1] ? context.options[1].allowMultiline : true;
```

Comment:
Should be ok since `allowMultiline` is the only option authorized by the rule schema.